src/components/analytics/HiringBySectionType.tsx

```diff
@@ -23,9 +20,6 @@ export const HiringBySectionType = ({ hireStreamName }: HiringBySectionTypeProps
 
     return (
         <>
-            <Text size="xl" style={{ marginTop: 10, marginLeft: 40 }}>
-                {tr('Hiring by section type')}
```

Comment:
Why delete the title? Multiple chart blocks could be placed on a single page?

Reply:
I fixed it